lib/livebook/secrets/secret.ex

```diff
@@ -25,5 +25,13 @@ defmodule Livebook.Secrets.Secret do
       message: "should contain only alphanumeric characters and underscore"
     )
     |> validate_required([:name, :value])
+    |> put_readonly()
+  end
+
+  defp put_readonly(changeset) do
+    case get_field(changeset, :hub_id) do
+      "enterprise" <> _ -> put_change(changeset, :readonly, true)
+      _any -> changeset
+    end
```

Comment:
This is not strictly related to this PR, right? In any case, I think we can remove this:

1. we will remove the enterprise stuff
2. teams won't have read-only secrets